app-editors/retext/retext-7.0.4-r1.ebuild

```diff
@@ -36,13 +36,18 @@ RDEPEND="
 	dev-python/markups[${PYTHON_USEDEP}]
 	dev-python/pygments[${PYTHON_USEDEP}]
 	dev-python/python-markdown-math[${PYTHON_USEDEP}]
-	dev-python/PyQt5[gui,network,printsupport,webengine,widgets,${PYTHON_USEDEP}]
+	dev-python/PyQt5[gui,network,printsupport,widgets,${PYTHON_USEDEP}]
```

Comment:
This block can be simplified though, right?

* Either more descriptive:
```
|| (
	(
		dev-python/PyQtWebEngine[${PYTHON_USEDEP}]
		>=dev-python/PyQt5-5.12[gui,network,printsupport,widgets,${PYTHON_USEDEP}]
	)
	dev-python/PyQt5[gui,network,printsupport,webengine,widgets,${PYTHON_USEDEP}]
)
```
* Or compact:
```
dev-python/PyQt5[gui,network,printsupport,widgets,${PYTHON_USEDEP}]
|| (
	dev-python/PyQtWebEngine[${PYTHON_USEDEP}]
	>=dev-python/PyQt5-5.12[webengine,${PYTHON_USEDEP}]
)
```

Please also note: One line per dependency.